zebra/zebra_srv6.c

```diff
@@ -214,7 +214,7 @@ void zebra_srv6_locator_format_set(struct srv6_locator *locator,
 
 	locator->sid_format = format;
 
-	if (IS_ZEBRA_DEBUG_PACKET)
```

Comment:
Should we change the documentation to say that now it's a separate command for SRv6 stuff?

Reply:
at least mention the presence of that command.
I don't see how we would  have to run a deprecation or something like that..